app/src/main/java/com/teamcatchme/calendar_customview/CalendarItemView.kt

```diff
@@ -0,0 +1,65 @@
+package com.teamcatchme.calendar_customview
+
+import android.content.Context
+import android.graphics.*
+import android.util.AttributeSet
+import android.util.Log
+import android.view.View
+import androidx.core.content.res.ResourcesCompat
+import com.teamcatchme.catchmesample.R
+
+class CalendarItemView @JvmOverloads
+constructor(
+    context: Context,
+    attrs: AttributeSet? = null,
+    defStyleAttr: Int = 0,
+    private val date: Int? = null,
+    private val catchuList: Array<Int> = arrayOf(),
+    private val isPrevious: Boolean = false
+) :
+    View(context, attrs, defStyleAttr) {
+    private var paint: Paint = Paint()
+
+    private fun drawDateRect(canvas: Canvas, date: Int) {
+        paint.textSize = 36f
+        if (isPrevious) paint.color = Color.GRAY
+        canvas.drawText(
+            date.toString(),
+            (width / 2).toFloat(),
+            ((height / 2) - ((paint.descent() + paint.ascent()) / 2)),
+            paint
+        )
+    }
+
+    private fun drawDateWithCatchuRect(canvas: Canvas, date: Int, catchuList: Array<Int>) {
+        paint.textSize = 30f
+        paint.color = Color.GRAY
+        val contextResources = context.resources
+        val catchuDrawable =
+            ResourcesCompat.getDrawable(contextResources, R.drawable.ic_cachu1, null);
+        val catchuBitmap = drawableToBitmap(requireNotNull(catchuDrawable))
+        canvas.drawBitmap(
+            requireNotNull(catchuBitmap),
+            (width / 2 - catchuBitmap.width / 2).toFloat(),
+            0f,
+            null
+        )
```

Comment:
```suggestion
        runCatching {
            canvas.drawBitmap(
                requireNotNull(catchuBitmap),
                (width / 2 - catchuBitmap.width / 2).toFloat(),
                0f,
                null
            )
        }.onFailure { Timber.e(it) } 
```

requireNotNull 같은 단정문이 있으면 익셉션이 터지기 때문에 방어로직을 하나 가져야할 것 같아